### ISSUE TITLE
upgrading from v6.3.0 to v6.4.0 breaks sequelize-cli

### ISSUE THREAD

**Issue by MichaelTaylor3D** [sequelize/cli#994]
Project works fine when migrating with v6.3.0, it breaks when using v6.4.0. I have tried using the `file://` scheme as it asks but it didnt make a difference in the error.

```
Sequelize CLI [Node: 16.0.0, CLI: 6.4.0, ORM: 6.13.0]

ERROR: Error reading "src\config\config.cjs". Error: Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only file and data URLs are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'
```

.sequalizerc
```
var path = require('path');

module.exports = {
  config: path.resolve('src', 'config', 'config.cjs'),
  'migrations-path': path.resolve('migrations'),
  'models-path': path.resolve('src', 'models'),
  'seeders-path': path.resolve('seeders'),
};
```

config.cjs
```
require('dotenv').config();

module.exports = {
  local: {
    dialect: 'sqlite',
    storage: './data.sqlite3',
  },
  test: {
    dialect: 'sqlite',
    storage: './test.sqlite3',
  },
  mirrorTest: {
    dialect: 'sqlite',
    storage: './testMirror.sqlite3',
  },
  mirror: {
    username: process.env.DB_USERNAME,
    password: process.env.DB_PASSWORD,
    database: process.env.DB_NAME,
    host: process.env.DB_HOST,
    dialect: 'mysql',
  },
};
```

**Comment by ephys** [comment]
Have you tried `path.resolve('./src', 'config', 'config.cjs')`?

**Comment by MichaelTaylor3D** [comment]
I just tried it and got the same error
![image](https://user-images.githubusercontent.com/5665004/149591644-d004ac86-31fc-4186-84bc-c4ce592dec4f.png)

This the './src' works as well in v6.3.0
![image](https://user-images.githubusercontent.com/5665004/149591735-1fef3346-d04c-4788-8b68-2936267727df.png)

**Comment by MichaelTaylor3D** [comment]
I added the `./src` but received the same error
![image](https://user-images.githubusercontent.com/5665004/149592042-85e213e2-f1a3-48e0-b250-2979c672b3ef.png)

the suggestion to use `./src` still works on v6.3.0
![image](https://user-images.githubusercontent.com/5665004/149592162-82baa612-b3be-4a6d-a023-894e0295e352.png)

**Comment by ephys** [comment]
So the issue is that `import()` expects paths that start either with `./` or `file://`. `C:\\` as `path.resolve` outputs isn't valid.

While we work on a solution, you'll need to specify the path without using path.resolve :/

**Comment by ephys** [comment]
Maybe `url.pathToFileURL(path.resolve('./src', 'config', 'config.cjs')).href` might work

**Comment by MichaelTaylor3D** [comment]
Was just about to post this, I attempted this before I reported the issue and received a different error: 
![image](https://user-images.githubusercontent.com/5665004/149592545-501222ad-e91a-4bba-b2c1-2e883748cad0.png)

**Comment by ephys** [comment]
Damn. We use path.resolve ourselves on it https://github.com/sequelize/cli/blob/main/src/helpers/config-helper.js#L46 which completely breaks your path.

I don't have a hotfix to give you, we need to fix that in the cli

**Comment by MichaelTaylor3D** [comment]
My solution for now is to downgrade to v6.3.0. But wanted to report this error with v6.4.0. If you would like any more information about the env, would be happy to post.

**Comment by ephys** [comment]
I think the fix is simple, I need replace this line
https://github.com/sequelize/cli/blob/main/src/helpers/import-helper.js#L24
with `await import(url.pathToFileURL(modulePath))`

It went unnoticed on UNIX because linux paths already look like URLs.

**Comment by ephys** [comment]
Managed to reproduce it and solution is as above. I'll open a PR

**Comment by WikiRik** [comment]
We published a new version (6.4.1) which should have fixed this issue. Let us know if it works or not. Thanks again for reporting it!

**Comment by MichaelTaylor3D** [comment]
v6.4.1 worked for me, thank you!
